src/stdlib_utils/parallelism_framework.py

```diff
@@ -35,6 +35,8 @@ class InfiniteLoopingParallelismMixIn:
         minimum_iteration_duration_seconds: In order for the process not to unnecessarily consume CPU resources while looping, the loop will sleep at the end of each iteration until this threshold duration is met.
     """
 
+    num_longest_iterations = 5
```

Comment:
what this variable is should be described in a docstring somewhere (probably the class docstring)